docs/src/sysimages.md

```diff
@@ -120,8 +120,8 @@ has been replaced, the next `create_sysimage` call will create a new sysimage
 based on the replaced sysimage. It is possible to create a sysimage
 non-incrementally by passing the `incremental=false` keyword. This will create
 a new system image from scratch. However, it will lose the special
-precompilation that the Julia bundled sysimage provides which is what make the
-REPL and package manager not require compilation after a Julia restart.. It is
+precompilation that the Julia bundled sysimage provides which is what makes the
+REPL and package manager to not require compilation after a Julia restart. It is
```

Comment:
This snippet reads more naturally to me without the additional `to`. Tangentially, perhaps add a comma between `provides` and `which`, i.e.
```suggestion
precompilation that the Julia bundled sysimage provides, which is what makes the
REPL and package manager not require compilation after a Julia restart. It is
```